airflow/www/static/js/dags.js

```diff
@@ -158,6 +162,9 @@ function lastDagRunsHandler(error, json) {
     const g = d3.select(`#last-run-${safeDagId}`);
 
     // Show last run as a link to the graph view
+    g.selectAll('a')
```

Comment:
Why do are we setting the graphUrl twice?

Reply:
I think I couldn't find a way to update the date without clearing it first.

Reply:
Let's combine them at least:

``` 
g.selectAll('a')
      .attr('href', `${graphUrl}?dag_id=${encodeURIComponent(dagId)}&execution_date=${encodeURIComponent(executionDate)}`)
      .html('')
      .insert(isoDateToTimeEl.bind(null, executionDate, { title: false }));```

Reply:
sure.